src/SparseVariationalApproximationModule.jl

```diff
@@ -278,14 +288,19 @@ inducing_points(f::ApproxPosteriorGP{<:SparseVariationalApproximation}) = f.appr
 #
 
 function API.approx_lml(
-    sva::SparseVariationalApproximation, l_fx::Union{FiniteGP,LatentFiniteGP}, ys; kwargs...
+    sva::AbstractSparseVariationalApproximation,
+    l_fx::Union{FiniteGP,LatentFiniteGP},
+    ys;
+    kwargs...,
 )
     return AbstractGPs.elbo(sva, l_fx, ys; kwargs...)
 end
 
+_get_prior(approx::SparseVariationalApproximation) = approx.fz.f
```

Comment:
How about just calling it 
```suggestion
prior(approx::SparseVariationalApproximation) = approx.fz.f
```
?
I think it shows up in a bunch more places (within AbstractGPs' VFE/DTC code too)...